Source/Core/UICommon/AutoUpdate.cpp

```diff
@@ -31,6 +31,7 @@
 
 namespace
 {
+bool s_update_triggered = false;
```

Comment:
(I was trying to see if we could avoid adding static state, but I guess spawning a process *is* modifying system global state anyway...)